ote/src/cljs/ote/views/transit_visualization/calendar.cljs

```diff
@@ -101,12 +101,15 @@
        [:div (stylefy/use-style (merge (style-base/flex-container "row")
                                        {:margin-top "1rem"
                                         :margin-bottom "1rem"}))
-        [ui/checkbox {:label "Näytä myös edellinen vuosi"
-                      :checked show-previous-year?
-                      :on-check #(e! (tv/->ToggleShowPreviousYear))}]
-        [ui/checkbox {:label "Näytä myös seuraava vuosi"
-                      :checked show-next-year?
-                      :on-check #(e! (tv/->ToggleShowNextYear))}]]]
+        [:div {:style {:flex 1}}
```

Comment:
Do we need a comment here "Width set via flex to avoid IE oveflowing element"? In case I touch this later :)
BTW Same could be accomplished using `width` property if flex was not involved.

Reply:
No we don't need. :)